Source/DafnyCore/AST/Expressions/Comprehensions/MapComprehension.cs

```diff
@@ -20,8 +20,8 @@ public MapComprehension(Cloner cloner, MapComprehension original) : base(cloner,
     Finite = original.Finite;
   }
 
-  public MapComprehension(IToken tok, RangeToken rangeToken, bool finite, List<BoundVar> bvars, Expression range, Expression/*?*/ termLeft, Expression termRight, Attributes attrs)
-    : base(tok, rangeToken, bvars, range, termRight, attrs) {
+  public MapComprehension(IOrigin tok, RangeToken rangeOrigin, bool finite, List<BoundVar> bvars, Expression range, Expression/*?*/ termLeft, Expression termRight, Attributes attrs)
```

Comment:
I'm puzzled as I would that thought that you'd rename `tok` to `origin` but not that you'd rename `rangeToken` to `rangeOrigin`

Anyway, these are parameter names so I presume they don't wieght a lot.

Reply:
I'll fix it in a follow-up. Hope that's OK.